src/main/java/com/bulletinboard/post/controller/PostController.java

```diff
@@ -0,0 +1,57 @@
+package com.bulletinboard.post.controller;
+
+import com.bulletinboard.post.dto.PostNewRequest;
+import com.bulletinboard.post.dto.PostResponse;
+import com.bulletinboard.post.dto.PostUpdateRequest;
+import com.bulletinboard.post.dto.PostUpdateResponse;
+import com.bulletinboard.post.service.PostService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+public class PostController {
+
+    private final PostService postService;
+
+    @PostMapping("/posts")
+    public ResponseEntity<PostResponse> savePost(@RequestBody PostNewRequest postNewRequest) {
+        Long savedPostId = postService.savePost(postNewRequest);
+        PostResponse postResponse = postService.findById(savedPostId);
+
+        return new ResponseEntity<>(postResponse, HttpStatus.OK);
+    }
+
+    @GetMapping("/posts")
+    public ResponseEntity<List<PostResponse>> findAllPosts() {
+        List<PostResponse> postResponses = postService.findAllPosts();
+
+        return new ResponseEntity<>(postResponses, HttpStatus.OK);
+    }
+
+    @GetMapping("/posts/{id}")
+    public ResponseEntity<PostResponse> findPostById(@PathVariable Long id) {
+        PostResponse postResponse = postService.findById(id);
+
+        return new ResponseEntity<>(postResponse, HttpStatus.OK);
+    }
+
+    @PostMapping("/posts/edit/{id}")
```

Comment:
```suggestion
    @PutMapping("/posts/{id}")
```
리소스를 수정할때에는 post가 아닌 put method가 더 적절해보여요.
그리고 path에서 edit을 생략하고 put method만으로 표현해보면 어떨까요?